src/map/zone_entities.cpp

```diff
@@ -1300,7 +1300,7 @@ void CZoneEntities::PushPacket(CBaseEntity* PEntity, GLOBAL_MESSAGE_TYPE message
             case CHAR_INRANGE_SELF: // NOTE!!!: This falls through to CHAR_INRANGE so both self and the local area get the packet
             {
                 TracyZoneCString("CHAR_INRANGE_SELF");
-                if (PEntity->objtype == TYPE_PC)
+                if (PEntity && PEntity->objtype == TYPE_PC)
```

Comment:
Theres part of a PEntity check in the GM section, but the PEntity check can go with or next to the packet check at the very start